forms-flow-web/src/formComponents/MinistrySelect/index.js

```diff
@@ -0,0 +1,210 @@
+import { Components, Formio } from 'react-formio';
+import _ from 'lodash';
+import { WEB_BASE_URL } from "../../apiManager/endpoints/config";
+import UserService from "../../services/UserService";
+
+const SelectComponent = Components.components.select;
+
+export default class MinistrySelect extends SelectComponent {
```

Comment:
I would suggest extending `RemoteSelect` here instead of `SelectComponent` so we don't have to duplicate the loadItems / updateItems methods. 

```javascript
export default class MinistrySelect extends RemoteSelect {
      static schema(...extend) {
       ...
      }
      static get builderInfo() {
              ...
      }
}
```